addons/l10n_mx/data/account_tax_data.xml

```diff
@@ -79,7 +79,7 @@
         <field name="amount">16</field>
         <field name="amount_type">percent</field>
         <field name="type_tax_use">sale</field>
-        <field name="tax_group_id" ref="tax_group_iva_16"/>
```

Comment:
Should we delete the data `tax_group_iva_16`?

Reply:
Nop @moylop260 

That data is for other taxes